### DISCUSSION TITLE
苹果手机扫描，扫码后提示禁止登陆，错误码45

### DISCUSSION THREAD

**Discussion by light-years-run** [lc-cn/onebots#114] [Q&A] [answered]
icqq协议，已经更新了最新版本qq
使用协议：Android_9.0.8.14755 [2024-01-23T10:48:12.849] [WARN] [icqq] - [禁止登录]登录失败，请前往QQ官网im.qq.com下载最新版QQ后重试，或通过问题反馈与我们联系。(错误码：45)

**Comment by MarryDream** [comment]
icqq 记得配置一下 qsign

**Comment by light-years-run** [comment]
> icqq 记得配置一下 qsign

是要搭建个qsign 服务吗，有人 风险挺高的，有风险不高的吗

**Comment by MarryDream** [comment, marked as answer]
> > icqq 记得配置一下 qsign
> 
> 是要搭建个qsign 服务吗，有人 风险挺高的，有风险不高的吗

用公用的风险相对高一点，自建自己用就安全很多

拿到链接后配置在这里就好
![image](https://github.com/lc-cn/onebots/assets/71262401/51f1f96d-9f34-465a-8e7d-e26d47ee3aaa)

**Comment by light-years-run** [comment]
ok 感谢
